components/pinecone/actions/delete-vectors/delete-vectors.mjs

```diff
@@ -0,0 +1,73 @@
+import app from "../../pinecone.app.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  key: "pinecone-delete-vectors",
+  name: "Delete Vectors",
+  description: "Deletes a vector by ID, from a single namespace. [See the docs](https://docs.pinecone.io/reference/delete_post).",
```

Comment:
Since the action takes an array of ids it might be interesting to have "vectors" instead of "a vector" here - what do you think?
```suggestion
  description: "Deletes one or more vectors by ID, from a single namespace. [See the docs](https://docs.pinecone.io/reference/delete_post).",
```